Practical-6/user-list-app/src/App.css

```diff
@@ -0,0 +1,50 @@
+.App {
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  background-color: rgb(211, 211, 211);
+  padding: 25px;
+  min-height: 100vh;
+}
+.user-list-main-container {
+  background-color: rgb(245, 245, 245);
+  max-width: 900px;
+  padding: 25px;
+  border: 1px;
+  border-radius: 25px;
+  box-shadow: 5px 5px 15px -8px;
+  display: flex;
+  flex-direction: row;
+}
+.user-list {
+  max-width: 620px;
+}
+.user-profile {
+  width: 200px;
+}
+@media screen and (max-width: 850px) {
+  .user-list-main-container {
+    display: flex;
```

Comment:
this class have already added display flex property no need to re-write if it's not changed